src/test/scala/uk/gov/nationalarchives/tdr/api/service/FileMetadataServiceSpec.scala

```diff
@@ -357,11 +369,42 @@ class FileMetadataServiceSpec extends AnyFlatSpec with MockitoSugar with Matcher
     testSetUp.statusTypeCaptor.getValue.sorted should equal(List(ClosureMetadata, DescriptiveMetadata))
   }
 
+  "addOrUpdateBulkFileMetadata" should "add or updated metadata and metadata statuses without validating the data when skipValidation is 'true'" in {
```

Comment:
Should also explicitly cover when skip validation = false

Reply:
We have an existing test covering this scenario however, I have updated the test method. Thanks